server/model/files.go

```diff
@@ -18,7 +18,7 @@ var (
 // Files struct is a row record of the files table in the tca database
 type Files struct {
 	File       int32          `gorm:"primary_key;AUTO_INCREMENT;column:file;type:int;" json:"file"`
-	Name       string         `gorm:"column:name;type:text;size:16777215;" json:"name"`
+	Name       string         `gorm:"index:idx_files_name,unique;column:name;type:text;size:16777215;" json:"name"`
```

Comment:
Just the name of the file probably isn't actually unique. The combination of the name and the path might be.

Reply:
(like there was a reason for the complexity in saveimage)

Reply:
I reverted this part.
I don't know how but I am pretty sure that I checked if the filename is indeed an unique characteristic (apparetnly not properly)